src/lib/pk_pad/iso9796/iso9796.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * ISO-9796-2 - Digital signature schemes giving message recovery schemes 2 and 3
+ * (C) 2016 Tobias Niemann, Hackmanit GmbH
+ *
+ * Botan is released under the Simplified BSD License (see license.txt)
+ */
+
+#ifndef ISO9796_H
```

Comment:
Header guard must match the format `BOTAN_XXX_H__` for the amalgamation script to work.